src/scripts/main.js

```diff
@@ -2,7 +2,41 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('tbody');
 
-// write your code here
+for (const human of people) {
+  const row = document.createElement('tr');
+  const humanName = document.createElement('td');
+  const humanGender = document.createElement('td');
+  const humanBorn = document.createElement('td');
+  const humanDied = document.createElement('td');
+  const humanAge = document.createElement('td');
+  const humanCen = document.createElement('td');
```

Comment:
let's do it using formatted strings + `insertAdjacentHTML` 😉 
so it'll be more clear